projects/core/src/model/misc.model.ts

```diff
@@ -128,4 +128,10 @@ export interface BaseSite {
   baseStore?: BaseStore;
   requiresAuthentication?: boolean;
   isolated?: boolean;
+  captchaConfig?: CaptchaConfig;
+}
+
+export interface CaptchaConfig {
+  enabled: boolean;
```

Comment:
Does it make sense `enabled` optional and assume truthy or falsy if it does not exist?

Reply:
According to the logic of backend commerce, it is either true or false, and it will not be null.